static/templates/about.html

```diff
@@ -89,12 +89,12 @@ <h2 class="amarante font-base">Our Tribal Council</h2>
         <li><a href="mailto:lceniceros@snohomishtribe.org">Treasurer: Lisa Ceniceros</a></li>
         <li><a href="mailto:kvansenus@snohomishtribe.org">Council Member 1: Kimber VanSenus</a></li>
         <li><a href="mailto:jchristoe@snohomishtribe.org">Council Member 2: Joel Christoe</a></li>
-        <li><a href="mailto:ebond@snohomishtribe.org">Council Member 3: Emily Bond</a></li>
+        <li><a href="mailto:bporter@snohomishtribe.org">Council Member 3: Becky Porter</a></li>
         <li><a href="mailto:tcarlson@snohomishtribe.org">Council Member 4: Tracey Carlson</a></li>
         <li><a href="mailto:ekielhorn@snohomishtribe.org">Council Member 5: Elijah Kielhorn</a></li>
         <li><a href="mailto:sworthington@snohomishtribe.org">Council Member 6: Salima Worthington</a></li>
-        <li><a href="mailto:bporter@snohomishtribe.org">Alternate 1: Becky Porter</a></li>
-        <li><a href="mailto:mchristoe@snohomishtribe.org">Alternate 2: Michael Christoe</a></li>
+        <li><a href="mailto:mchristoe@snohomishtribe.org">Alternate 1: Michael Christoe</a></li>
+        <li><a href="#">Alternate 2: Open</a></li>
```

Comment:
I would make this open spot not a link, since there's nothing to click. So just `<li>Alternate 2: Open</li>`.